Source/FFmpegMediaSource.h

```diff
@@ -239,7 +239,7 @@ namespace winrt::FFmpegInteropX::implementation
 
     private:
 
-        winrt::weak_ref<MediaStreamSource> mssWeak = { nullptr };
+        MediaStreamSource mss{ nullptr };
```

Comment:
For workflows using the `CreateFrom*` factory method, there is no proper owner of the "strong" `MediaStreamSource` instance, thus the reference count would go to 0 as soon as `CreateMediaStreamSource()` exited.

In these cases, `FFmpegMediaSource` must own the strong reference.